src/zkevm_specs/evm/table.py

```diff
@@ -289,6 +289,15 @@ class CopyDataTypeTag(IntEnum):
     TxCalldata = auto()
     TxLog = auto()
 
+    # RLC Accumulator tag can be used whenever we wish to
+    # accumulates `value` iteratively over all the steps in
+    # copy circuit. This is specifically used in the SHA3
+    # opcode execution where the copy table's last row has
+    # an accumulated value that is the RLC representation of
+    # all input bytes. Using this value, we can then lookup
+    # the Keccak table for the SHA3 of the input bytes.
+    RlcAcc = auto()
```

Comment:
would you like to update copy-table in  `zkevm-specs/specs/tables.md `with new added field ?  maybe `zkevm-specs/specs/copy-proof.md` as well ?

Reply:
Added :)